spec/listformat.html

```diff
@@ -77,7 +77,8 @@ <h1>CreatePartsFromList ( _listFormat_, _list_ )</h1>
           1. Let _placeables_ be a new Record { [[0]]: _first_, [[1]]: _second_ }.
           1. Return DeconstructPattern(_pattern_, _placeables_).
         1. Let _last_ be a new Record { [[Type]]: `"element"`, [[Value]]: _list_[_size_ - 1] }.
-        1. Let _parts_ be &laquo; _last_ &raquo;.
+        1. Let _parts_ be ArrayCreate(0).
+        1. Append _last_ as the last element of _parts_. 
```

Comment:
You can't just "append" to an `Array` object, let alone put a Record in an Array. Let's revert this change, and do only the change below.

Reply:
Sorry I'm not sure what the correct syntax here should be. Removing `tail` is not sufficient because the type of `parts` would be either `Array` or `Record` and in `FormatList` it expects it to always be an `Array`.

Reply:
I don't understand. FormatList very much does not want an `Array`, since it does "For each part in parts, do", which is not well-defined for an `Array`.

Reply:
ah kk that's probably where I'm misunderstanding then. Can you clarify how `For each part in parts, do` is supposed to be implemented? Thanks!

Reply:
That will depend entirely on your internal implementation of the List specification type.

Reply:
gotcha. Ok then that makes sense. I'll make the change below then